src/clj/rems/api/forms.clj

```diff
@@ -15,6 +16,19 @@
    :end (s/maybe DateTime)
    :active s/Bool})
 
+(s/defschema FormField
+  {:title {s/Keyword s/Str}
+   :optional s/Bool
+   :type (s/enum "attachment" "date" "description" "label" "multiselect" "option" "text" "texta")
+   (s/optional-key :maxlength) (s/maybe (s/constrained s/Int (comp not neg?)))
+   (s/optional-key :options) [{:key s/Str
+                               :label {s/Keyword s/Str}}]
+   (s/optional-key :input-prompt) {s/Keyword s/Str}})
+
+(s/defschema FullForm
+  (merge Form
+         {:fields [s/Any]}))
```

Comment:
not `[FormField]`?

Reply:
That was my initial plan but there is a ton of fields not in use elsewhere.